### PR TITLE
[xc-admin] improve UI/UX of xc-admin-frontend

### PR DESCRIPTION
**Pull request by cctdaniel**
- fetch all proposals instructions and pass to individual proposal row
- add new verified icon
- enable scroll to top when clicking on individual proposal
- split devnet/pythtest and mainnet-beta/pythnet proposals
- show symbols for product and price account when possible

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx [hunk 1]
```diff
@@ -843,14 +843,111 @@ const Proposal = ({
...
+    squads,
     priceFeedMultisigAccount,
     priceFeedMultisigProposals,
     isLoading: isMultisigLoading,
   } = useMultisigContext()
 
+  useEffect(() => {
+    const fetchAllProposalsInstructions = async () => {
+      if (squads) {
+        const allProposalsIxs = await getManyProposalsInstructions(
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:869]: Should this call be in useMultisig

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx [hunk 2]
```diff
@@ -843,14 +843,111 @@ const Proposal = ({
...
+        )
+        const proposalsRes: TransactionAccount[] = []
+        const instructionsRes: MultisigInstruction[][] = []
+        // filter proposals for respective devnet/pythtest and mainnet-beta/pythnet clusters
+        parsedAllProposalsIxs.map((ixs, idx) => {
+          // pythtest/pythnet proposals
+          if (
+            isRemoteCluster(cluster) &&
+            ixs.length > 0 &&
+            ixs.every((ix) => ix instanceof WormholeMultisigInstruction)
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:893]: If you want to be precise maybe something more like : 
`(ix instanceof WormholeMultisigInstruction) && (ix.name == "postMessage") && (ix.governanceAction instanceof ExecutePostedVaa ) &&  (ix.governanceAction.targetChainId === 'pythnet')`

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx [hunk 3]
```diff
@@ -129,82 +133,86 @@ const StatusTag = ({ proposalStatus }: { proposalStatus: string }) => {
...
+  return (
+    <div className="flex items-center">
+      <Tooltip.Provider delayDuration={100} skipDelayDuration={500}>
+        <Tooltip.Root>
+          <Tooltip.Trigger>
+            <VerifiedIcon />
+          </Tooltip.Trigger>
+          <Tooltip.Content side="top" sideOffset={8}>
+            <span className="inline-block bg-darkGray3 p-2 text-xs text-light hoverable:bg-darkGray">
+              The instructions in this proposals are verified.
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:146]: `proposals` -> `proposal`

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx [hunk 4]
```diff
@@ -129,82 +133,86 @@ const StatusTag = ({ proposalStatus }: { proposalStatus: string }) => {
...
+  mapping: { [key: string]: string }
+  title: string
+  pubkey: string
+}) => {
+  return (
+    <div className="flex justify-between pb-3">
+      <div className="max-w-[80px] break-words sm:max-w-none sm:break-normal">
+        &#10551; {title}
+      </div>
+      <div className="space-y-2 sm:flex sm:space-x-2">{mapping[pubkey]}</div>
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:169]: Maybe make this bold. 
We should try to make the symbol here and the name of the instruction "addPub" more visible

**Review by guibescos — APPROVED** [review]

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx (hunk 4 above)
Comment (cctdaniel) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:169]: im not too keen on arbitrarily bolding certain variables here and not the others -- but note taken on making it more visible

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx (hunk 2 above)
Comment (cctdaniel) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:893]: the purpose of this was to show all Wormhole proposals and those that are not verified will be easily distinguishable through the verified checks

Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:893]: I you want to show all the proposals (it's fine) you need to come up with better logic. 
Right now, if a proposal has both a WormholeInstruction and a PythInstruction it won't show anywhere

Comment (cctdaniel) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx:893]: is that possible though?

**Pull request merged**